eval/eval.go

```diff
@@ -408,7 +408,12 @@ func (e *evalContext) evaluateImports() {
 //
 // Each environment in the import closure is only evaluated once.
 func (e *evalContext) evaluateImport(myImports map[string]*value, decl *ast.ImportDecl) {
-	name := decl.Environment.GetValue()
+	// If the import does not have a name, there's nothing we can do. This can happen for environments
+	// with parse errors.
+	if decl.Environment == nil {
+		return
+	}
+	name := decl.Environment.Value
```

Comment:
Is the change from `decl.Environment.GetValue()` semantically different than the new `decl.Environment.Value`?

Reply:
`decl.Environment.GetValue()` returns the empty string if `decl.Environment` is `nil`. So there is a semantic difference here in that prior to these changes we would continue attempting to evaluate the import. With these changes we will not. There is no change in semantics for non-`nil` values of `decl.Environment`.